package.json

```diff
@@ -175,7 +175,7 @@
     "react-pdf": "^4.0.5",
     "react-popper": "^2.2.3",
     "react-remove-scroll": "^2.4.0",
-    "react-select": "^4.3.0",
+    "react-select": "^5.4.0",
```

Comment:
To be checked because there is a few BC in the lib, no? https://react-select.com/upgrade

Reply:
We may be impacted by the changes on `ref`s since we indeed use one on `ReactSelect` in `SelectBox` to measure some height. Further investigation is needed for it. The remaining BCs look ok for us.

Reply:
This brings types without needing to install a `@types` package:
> Convert to TypeScript - TypeScript types now come packaged with react-select so you no longer need to have \@types/react-select installed; we no longer include Flow types

Reply:
J'ai finalement retiré la montée de version de ce paquet et installé `@types/react-select`, ça évite d'avoir à gérer le breaking change dans cette PR, et ça évite d'avoir à mettre à jour les snapshots.